pkg/controllers/node/ipam.go

```diff
@@ -137,7 +158,7 @@ func (c *NodeController) syncIPAMCleanup() error {
 
```

Comment:
Shouldn't we be setting canDelete to false at line 135?

Reply:
Yep I think so!